client/src/components/cardClosedTrade.js

```diff
@@ -12,22 +12,22 @@ import CardImageView from '../microcomponents/cardImageView'
 
 var colorMap = {
     P: {
-        title: "#47AFCB",
+        title: "#52B9D1",
```

Comment:
Think about using [styled components](https://www.styled-components.com/) if you need to do alot of this